index.html

```diff
@@ -6,5 +6,9 @@
     <link rel="stylesheet" type="text/css" href="styles.css" />
     <title>Quiz App</title>
   </head>
-  <body></body>
+  <body>
```

Comment:
Great job. I would recommend adding a header above your body element for accessibility. You could put your h1 into your header :)